doc/PFB/README.md

```diff
@@ -0,0 +1,238 @@
+# Portable Format for Biomedical Data
```

Comment:
why are these 4 PFBs files in the cloud-automation doc? can we move them to the pypfb repo doc, or leave them in cdis-wiki?